SIGs.md

```diff
@@ -0,0 +1,49 @@
+# Special Interest Groups (SIGs)
+
+Semantic conventions in OpenTelemetry are organized into distinct areas,
+with each area being overseen by a Special Interest Group (SIG). A SIG is
+composed of multidisciplinary contributors who bring expertise in both the
+relevant domain and OpenTelemetry itself.
+
+SIGs have several responsibilities and goals, but their primary focus is to
+define and guide semantic conventions toward stabilization. They are also tasked
+with the long-term maintenance of these conventions, serving as domain experts
+for pull request (PR) reviews and issue triage.
```

Comment:
maybe something around these lines?
There is a push in the commuity repo to have smaller, well-scoped projects and permanent SIGs are seen more as an exception than a common scenario. E.g. we had HTTP or JVM  SIG but they had achieve their goals and are dissolved now.

```suggestion
define and guide semantic conventions toward stabilization. SIGs that achieve their goal 
may be dissolved, but the codeowner group stays serving as domain experts
for pull request (PR) reviews and issue triage.
```